src/ecdsa/test_pyecdsa.py

```diff
@@ -410,6 +418,10 @@ def test_from_openssl_brainpoolp512r1(self):
     def do_test_from_openssl(self, curve):
         curvename = curve.openssl_name
         assert curvename
+
+        if curvename not in OpenSSL.supported_curves:
+            return
+
```

Comment:
this will cause the test to be reported as passing instead of skipped, please use [pytest mechanism](https://docs.pytest.org/en/latest/skipping.html) to skip the test case

Reply:
the current code is using decorators:
https://github.com/warner/python-ecdsa/blob/e29ddec6042450335cb0490e8a53e440ab7f02c5/src/ecdsa/test_pyecdsa.py#L739-L740